backend/settings.py

```diff
@@ -134,4 +135,27 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
-DEFAULT_FILE_STORAGE = 'canvas_app_explorer.storage_get_file.DatabaseFileStorage'
\ No newline at end of file
+DEFAULT_FILE_STORAGE = 'canvas_app_explorer.storage_get_file.DatabaseFileStorage'
+
+# TODO: Switch this to CSP for additional security
+X_FRAME_OPTIONS = 'ALLOWALL'
+
+# So request works over the proxy
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+DB_CACHE_CONFIGS = os.getenv('DB_CACHE_CONFIGS',
+                           {'CACHE_TTL': 600, 'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
+                            'LOCATION': 'django_myla_cache',
```

Comment:
still says django_myla_cache here

Reply:
Thanks! I copied some of this from myla, I'll update this name.